e2e_test/batch/aggregate/sort_agg.slt.part

```diff
@@ -11,24 +11,25 @@ statement ok
 create materialized view mv as select * from t order by v1 desc
 
 query I rowsort
-select max(v2), v1 from mv group by v1
+select v1, max(v2) from mv group by v1
 ----
-10 100
-20 10
-3 9
-7 4
+-10 100
 1 1
-100 -10
+10 20
+100 10
+4 7
+9 3
+
```

Comment:
@st1page I fixed the order field of the batch_sort_agg, which messed up this test case again. Maybe I misuse rowsort here, but I would expect the output to be sorted by v1 desc.

Reply:
I think we must use `rowsort` here and the output is not expected to be sorted. Because even if we use sortAgg here, the Exchange without order required will reorder the records.